### PR TITLE
Cba Medical Updated

### PR DESCRIPTION
**Pull request by micahwhitsel**
**When merged this pull request will:**
- Describe what this pull request will do
- Each change in a separate line
- Include documentation if applicable
- Respect the [Development Guidelines](https://github.com/7Cav/cScripts/wiki/Code-and-development-policy)

### CODE REVIEW

**Comment by micahwhitsel** [comment]
There have been issues with the medical systems because of another ACE update. ace_medical_treatment_advancedBandages is no longer boolean

File: cba_settings.sqf [hunk 1]
```diff
@@ -216,8 +217,8 @@ force force ace_medical_treatment_cprSuccessChance = 0.5;
 force force ace_medical_treatment_holsterRequired = 0;
 //ace_medical_treatment_litterCleanupDelay = 600;
 force force ace_medical_treatment_locationEpinephrine = 0;
-force force ace_medical_treatment_locationPAK = 3;
-force force ace_medical_treatment_locationsBoostTraining = true;
+force force ace_medical_treatment_locationPAK = 0;
+force force ace_medical_treatment_locationsBoostTraining = false;
```
Comment (ColdEvul) [inline cba_settings.sqf]:
```suggestion
force force ace_medical_treatment_locationsBoostTraining = true;
```

[the remark this answers is not in the document]
Comment (micahwhitsel) [reply cba_settings.sqf]: Yes, apparently this was never approved at the battalion level. We just want PAKing everywhere for the medics

[the remark this answers is not in the document]
Comment (ColdEvul) [reply cba_settings.sqf]: Ok that one is good cause the later was retarted. Tho the question is regarding the location boost

[the remark this answers is not in the document]
Comment (micahwhitsel) [reply cba_settings.sqf]: With location boost on platoon medics can pak and troopers can stitch near a medical building or vehicle and that's not something we're looking to have

[the remark this answers is not in the document]
Comment (ColdEvul) [reply cba_settings.sqf]: It +1 on all levels so trooper become medics and medics become doctors.
Its a emergency good thing when a situation arizes and a fieald hostpital is establioshed

[the remark this answers is not in the document]
Comment (micahwhitsel) [reply cba_settings.sqf]: Ok, think I just committed the suggestion

**Pull request merged**